### ISSUE TITLE
Examples in the documentation

### ISSUE THREAD

**Issue by AntonioCarta** [ContinualAI/avalanche#886]
We should add some examples in the documentation, at least for the high level classes and all the features more frequently used by the end user.

See #885 for an example

**Comment by niniack** [comment]
Is the idea that an Avalanche documentation page such as [this about the SimpleCNN class](https://avalanche-api.continualai.org/en/v0.1.0/generated/avalanche.models.SimpleCNN.html#avalanche.models.SimpleCNN) would include an example [similar to this](https://pytorch.org/docs/stable/generated/torch.nn.Conv1d.html#:~:text=%3E%3E%3E%20m%20%3D%20nn.conv1d(16%2C%2033%2C%203%2C%20stride%3D2)) 

If I have got the idea, then I'm interested in contributing! If there is anything I should keep in mind to get started, please let me know.

**Comment by AntonioCarta** [comment]
Yes, that's what we want to achieve. I think you can start immediately, you don't need much background on avalanche to do it. You can make a PR once you have a couple of examples if you want some comments.
